web_development_101/the_backend/database_basics_lesson.md

```diff
@@ -4,7 +4,7 @@ We've talked about the client-side and the server-side but how do we keep ahold
 
 Databases are kind of hidden in the back of the web application so people treat them with a sort of suspicion and awe.  **That's nonsense and you should get over it** -- your database and you are going to become very good friends (or at least frenemies).  By the end of this curriculum, you're going to understand what's going on with your databases and be able to interact with them like a pro (and probably better than some people you'll work with).  This lesson is a teaser for that.
 
-Compared to a normal programming language like you've already learned, SQL (Structured Query Language), which is used to query databases, is a very simple syntax... there are only a small handful of verbs to learn.  What trips people up is that you need to be able to visualize in your head what it's going to be doing.  We'll spend a fair bit of time on SQL and databases because they're so fundamental, but for now we'll just cover enough to get you familiar with what's going on in there.
+Compared to a normal programming language like you've already learned, SQL (Structured Query Language), which is used to query databases, is a very simple syntax... there are only a small handful of verbs to learn.  What trips people up is that you need to be able to visualize in your head what it's going to happen.  We'll spend a fair bit of time on SQL and databases because they're so fundamental, but for now we'll just cover enough to get you familiar with what's going on in there.
```

Comment:
this sentence makes more sense with "to be doing" I don't think its a typo

Reply:
ok